student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, _classroom, name = 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @classroom = nil
+  end
+
+  def play_hook
+    '¯\(ツ)/¯'
+  end
+end
```

Comment:
- [ ] in the Student class, there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom). have a look at [the article with examples](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)